docs/GraFx-Studio/concepts/design-run/index.md

```diff
@@ -0,0 +1,63 @@
+# Design Mode and Run Mode
+
+GraFx Studio provides two modes for creating and testing Smart Templates: **Design Mode** and **Run Mode**. 
+
+These modes allow you to focus separately on designing the smart template and testing business logic.
+
+In **Design Mode**, you can freely arrange and style elements without triggering actions or interactions. This keeps the workspace clean, ideal for setting up and adjusting layouts, styles, and initial logic.
+
+Switching to **Run Mode** simulates the end-user experience (My Projects), showing how the template’s logic behaves live. In this mode, actions and visibility settings behave as they will in a real environment, letting you test how users will experience the template.
+
+## Design Mode
+
+**Design Mode** is optimized for layout and style tasks, with logic and actions inactive.
+
+![screenshot](dr1.png)
+
+![screenshot-full](dr1_1.png)
+
+### Key Features in Design Mode:
+- **Static Layout**: Actions and logic are inactive, letting you set up without interference.
+- **Focused Styling**: Only layout and style elements are active, making design adjustments easier.
+  
+### Elements Active in Design Mode:
+- **Anchoring**: Set relative positions to ensure consistent layout across sizes.
+- **Animation Setup**: Define and tes animations with playback
+- **Styling**: Customize text, colors, and layout freely.
+
+## Run Mode
+
+In **Run Mode**, the template operates as it would for end users, activating logic and interactions for realistic testing.
+
+![screenshot](dr2.png)
+
+![screenshot-full](dr2-1.png)
+
+### Key Features in Run Mode:
+- **Live Actions**: Business logic is triggered in response to changes, reflecting end-user interactions.
+- **True Visibility**: Hidden variables and conditional visibility behave as configured for end users.
```

Comment:
With "conditional visibility" you mean with actions? Because we don't have visibility conditions yet.

And I would also mention required variables because in Design mode they can be left empty. In Run mode they must have a value.

Reply:
I refer to the layers and variables. I'll make it more clear